src/main/webapp/frontend/styles/style.css

```diff
@@ -0,0 +1,3 @@
+.error-notification {
+  color: red;
```

Comment:
keine Border-color?

Reply:
ds heimer hüt besproche z üs druf geiniget dasmers nid mache

Reply:
Grund dafür: Nicht möglich in Vaadin eine zusätzliche CSS-Angabe für die Notifikation zu setzen. Entweder wir färben alle Notifications rot, oder keine.

Reply:
-- Die Border-Color hier zu setzen führt zu einem doppelten Rand. 😉

Reply:
Kann man nicht für die normalen Notifications eine eigene Klasse erstellen, welche ein anderes CSS hat?

Reply:
> -- Die Border-Color hier zu setzen führt zu einem doppelten Rand. 😉

`border-style: solid;`?